ravencoin/types.go

```diff
@@ -38,9 +38,9 @@ const (
 	// used in Currency.
 	Decimals = 8
 
-	// SatoshisInBitcoin is the number of
+	// SatoshisInRavencoin is the number of
 	// Satoshis in 1 BTC (10^8).
-	SatoshisInBitcoin = 100000000
+	SatoshisInRavencoin = 100000000
```

Comment:
Idk if this is used anywhere or if changing this value will break anything but it's wrong (unfortunately 😭)

Reply:
I thought we still used 10^8 as the number of satoshis in 1 RVN.

Reply:
yes, it's my bad, they become RVN satoshis so it works. In my head I was thinking about the value of 1 RVN in BTC satoshis, which ofc isn't 10^8

Reply:
No worries. Yea, the whole "satoshis" unit isn't the most straight forward.

Reply:
https://github.com/RavenProject/Ravencoin/blob/9def8ed6b5aa6f8d6aa3a53436197233941d7e17/src/amount.h#L15